slidez-core/src/main/resources/new_database/script.sql

```diff
@@ -0,0 +1,116 @@
+CREATE TABLE IF NOT EXISTS "user" (
```

Comment:
again, generating DB from zero. We need to update existing migrations, not re-write them from scratch